shared/mortgage.js

```diff
@@ -72,3 +80,87 @@ export function getActiveMortgageByBorrower(
     mortgage => mortgage && mortgage.borrower === borrower
   )
 }
+
+/**
+ * Computes the punitory and non-punitory interest of a given mortgage and returns
+ * the outstanding amount
+ * @param {mortgage} - Mortgage to compute interest
+ * @returns {number} - Mortgage Outstading amount
+ */
+
+export function getMortgageOutstandingAmount(mortgage) {
+  const now = parseInt(new Date().getTime() / 1000, 10)
```

Comment:
if mortgage is required, maybe we can check here if it exists and return 0 otherwise, relieving the caller from doing so